CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 3.7.1 (2019-07-02)
+
+- Fixed `skyux build` command to ignore SPA's `fixtures` directories. [#136](https://github.com/blackbaud/skyux-sdk-builder/pull/136)
+- Fixed package dependencies to install missing `ajv` package. [#131](https://github.com/blackbaud/skyux-sdk-builder/pull/131)
+- Fixed invalid import statement generated by `skyux generate component` command. [#135](https://github.com/blackbaud/skyux-sdk-builder/pull/135)
```

Comment:
Fixed the invalid import statement generated by the `skyux generate component` command.